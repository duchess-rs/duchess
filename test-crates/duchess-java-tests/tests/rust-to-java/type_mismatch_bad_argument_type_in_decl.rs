//@compile-flags: --crate-type lib

duchess::java_package! {
    package type_mismatch;

    public class TakesInt {
        //~^ ERROR: method `take(long)` does not match any of the methods in the reflected class
        public void take(short);
    }
}

fn main() {}
