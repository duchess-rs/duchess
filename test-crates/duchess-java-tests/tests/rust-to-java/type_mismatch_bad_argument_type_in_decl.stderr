error: error in class `type_mismatch.TakesInt`: method `take(long)` does not match any of the methods in the reflected class: take(int)
 --> tests/ui/type_mismatch_bad_argument_type_in_decl.rs:6:5
  |
6 |     public class TakesInt {
  |     ^^^^^^

error: aborting due to 1 previous error

